### DISCUSSION TITLE
Need design feedback for new song select footer(V2)

### DISCUSSION THREAD

**Discussion by tomm13** [ppy/osu#28159] [Polls]
| Comparison | Without glow (before) | With glow (after) |
| :--- | :---: | :---: |
| Autopilot  | ![image](https://github.com/ppy/osu/assets/42684333/653051a5-9d17-416d-a86e-254ac41656e7)|![image](https://github.com/ppy/osu/assets/42684333/b9e7d7fc-adfd-4768-8348-be28c341f2d3)|
| Increased difficulty | ![image](https://github.com/ppy/osu/assets/42684333/46c22b5c-c3b9-45e2-bccf-6fa83960c78e) |![image](https://github.com/ppy/osu/assets/42684333/aee8bfb4-7800-40de-b71a-73485f08a792)|
| Decreased difficulty | ![image](https://github.com/ppy/osu/assets/42684333/90f26fc1-dbbb-4b14-b06e-173280fe298a)|![image](https://github.com/ppy/osu/assets/42684333/0a248216-0c3c-4c63-a6ed-59d2122b0ac6)|

Ref:
![image](https://github.com/ppy/osu/assets/42684333/679c7ca4-54da-47a4-83d5-a412a2a8b719)

**Comment by peppy** [comment]
I think it's too much.
